### PR TITLE
#56 Replace Pebble with Handlebars

### PR DESCRIPTION
**Pull request by lukasz-gryzbon**

### CODE REVIEW

File: src/test/java/com/lsd/report/approval/LsdContextTest.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ void createsReportWithScenariosAndEvents() {
         lsdContext.addFact("some important value", "123456");
         lsdContext.completeScenario("An Error scenario", "<p>Failure! Expected value to be 123 but was 123456</p>", ERROR);
 
-        Approvals.verify(lsdContext.completeReport("Approval Report").toFile());
+        Approvals.verifyHtml(lsdContext.generateReport("Approval Report"));
```
Comment (lukasz-gryzbon) [inline src/test/java/com/lsd/report/approval/LsdContextTest.java:72]: @nickmcdowall Are you OK with this change? And same below...

Comment (nickmcdowall) [reply src/test/java/com/lsd/report/approval/LsdContextTest.java:72]: Hey, I recall the generateReport was added ages ago but was it just for testing purposes (can't remember). I guess it should be fine but maybe the completeReport method needs changing so we don't need to expose other public methods to get the report html.. but let's go ahead for now until that can be figured out I guess.

**Review by nickmcdowall — APPROVED** [review]
Looks good  👍  (added comment to your query).

File: src/test/java/com/lsd/report/approval/LsdContextTest.java (hunk 1 above)
Comment (lukasz-gryzbon) [reply src/test/java/com/lsd/report/approval/LsdContextTest.java:72]: I think it was added for the lsd-ui project, not only for testing.

**Pull request merged**